src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceMultiSource.java

```diff
@@ -53,6 +54,16 @@ public ReferenceMultiSource( final PipelineOptions pipelineOptions, final String
         this.referenceWindowFunction = referenceWindowFunction;
     }
 
+    /**
+     * @param auth authentication information
+     * @param referenceURL the name of the reference (if using the Google Genomics API), or a path to the reference file
+     * @param referenceWindowFunction the custom reference window function used to map reads to desired reference bases
+     */
+    public ReferenceMultiSource(final AuthHolder auth, final String referenceURL,
+                                   final SerializableFunction<GATKRead, SimpleInterval> referenceWindowFunction) {
+        this(auth.asPipelineOptionsOldStyle(), referenceURL, referenceWindowFunction);
```

Comment:
Since time-related information like `OldStyle` embedded in method/class names tends to get out-of-date quickly, recommend renaming to just `asPipelineOptions()`